### ISSUE TITLE
Question about the number of output_projection in Beit3

### ISSUE THREAD

**Issue by violet-sto** [microsoft/unilm#1078]
**Describe**
Beit3:

Hi!

I found that there is only one output_projection (nn.Linear(768, 64000)) for masked language modeling. However, as Beit-3 is a multimodal model, should there also be a output head for masked image modeling?

**Comment by wenhui0924** [comment]
Hi @violet-sto,

The head for masked language modeling is "mim_head.weight" and "mim_head.bias". They are in the [BEiT3-Large](https://conversationhub.blob.core.windows.net/beit-share-public/beit3/pretraining/beit3_large_patch16_224.pth) and [BEiT3-Base](https://conversationhub.blob.core.windows.net/beit-share-public/beit3/pretraining/beit3_base_patch16_224.pth)

**Comment by violet-sto** [comment]
Thank you so much for your reply! Unfortunately, I didn't find where 'mim_head' was defined in the code of Torchscale.
